main.rb

```diff
@@ -1,159 +1,168 @@
 require_relative 'app'
 
-def main
-  app = App.new
-  running = true
-  while running
-    print "1. List all books\n2. List all people\n3. Create a person\n4. Create a book\n5. Create a rental\n6. List all rentals for a given person id\n7. Exit\n"
+def book_list(app)
+  print "Book list:\n"
+  if app.books.length.positive?
+    app.list_books
+    print 'Press enter to continue'
+  else
+    print 'There are not any books added yet'
+  end
+  gets.chomp
+end
 
-    print "\nSelect an option by number: "
+def people_list(app)
+  if app.people_exist?
+    app.list_people
+    print "\nPress enter to continue\n"
+  else
+    print "There are not any people added yet\n"
```

Comment:
- [ ] Kindly ensure that your `Main.rb` file contains code that is responsible for starting your console app and asking the user for the initial choice of options or to quit. You can achieve this by extracting some pieces of logic to separate files or modules. This will enhance cleaner code.